org.tribler.Tribler.metainfo.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally this should be submitted and maintained upstream.

Reply:
Agreed! I'll propose it, but I'm not part of tribler.